lib/v2/snow-peak/new-arrivals.js

```diff
@@ -0,0 +1,41 @@
+const got = require('@/utils/got');
+const { art } = require('@/utils/render');
+const path = require('path');
+const cheerio = require('cheerio');
+const host = 'https://www.snowpeak.com';
+module.exports = async (ctx) => {
+    const url = `${host}/collections/new-arrivals`;
+    const response = await got({
+        method: 'get',
+        url,
+    });
+    const data = response.data;
+
+    const $ = cheerio.load(data);
+    const list = $('.element.product-tile')
+        .map(function () {
+            const data = {};
+            const details = $(this).find('.product-data').eq(0).attr('data-product');
+            const product = JSON.parse(details);
+            data.title = product.title;
+            const imgUrl = product.featured_image;
```

Comment:
I would prefer to show all the images

Reply:
some products have up to ten images, so maybe it is not a good idea.